packages/kbn-es-query/src/utils/__tests__/get_time_zone_from_settings.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// TINA WIP
+import expect from '@kbn/expect';
+import moment from 'moment-timezone';
+import { getTimeZoneFromSettings } from '../get_time_zone_from_settings';
+
+// following similar methodology to kibana/src/legacy/core_plugins/kibana/common/field_formats/types/__tests__/date.js
+
+// eslint-disable-next-line mocha/no-exclusive-tests
+describe('get timezone from settings', function () {
+  let mockConfig;
+
+  beforeEach(function () {
+    mockConfig = {};
+    mockConfig.dateFormatTZ = 'Browser';
+  });
+
+  it('should return the system timezone if the time zone is set to "Browser"', function () {
+    function setDefaultTimezone() {
+      moment.tz.setDefault(mockConfig.dateFormatTZ);
+    }
+    mockConfig.dateFormatTZ = 'America/Chicago';
+    setDefaultTimezone();
+
+    const result = getTimeZoneFromSettings(mockConfig.dateFormatTZ);
+    expect(result).to.eql(mockConfig.dateFormatTZ);
+  });
+
+  it('should return the config timezone if the time zone is set', function () {
```

Comment:
I think the descriptions and the implementations between this test and the one above got flip flopped somehow?

It seems like this would be simpler to read without the helper function. 

I would just do something like this:
```
moment.tz.setDefault('America/New York');
const result = getTimeZoneFromSettings('Browser');
expect(result).to.equal('America/New York');
```

Also you'll need to call `moment.tz.setDefault()` in Mocha's `afterEach` hook to reset moment back to it's original default value, otherwise the change to its global state could affect other tests.